### ISSUE TITLE
Slashes kill screenshots

### ISSUE THREAD

**Issue by mrbisson-ibm** [gkushang/cucumber-html-reporter#45]
When writing filenames for screenshots, characters which are illegal in paths which are found in step names can kill the report generation. E.g. I got this error

```
ENOENT, no such file or directory 'reports/screenshot/I_open_file_"/public/foo"_2043.png'
```

**Comment by gkushang** [comment]
@mrbisson-ibm Thanks for reporting and fixing it! Published the fix with `v0.3.7`

**Comment by mrbisson-ibm** [comment]
Thanks @gkushang! :-)
